ci-operator/config/openshift/cluster-api-provider-kubevirt/openshift-cluster-api-provider-kubevirt-master.yaml

```
@@ -0,0 +1,42 @@
+base_images:
+  base:
+    name: "4.6"
```

Unfortunately there was a branch cut in the meantime, so masters are now 4.7.
```suggestion
    name: "4.7"
```